eslint.config.mjs

```diff
@@ -0,0 +1,12 @@
+import pluginJs from "@eslint/js";
+import globals from "globals";
+import tseslint from "typescript-eslint";
+
+/** @type {import('eslint').Linter.Config[]} */
+export default [
+  { files: ["**/*.{js,mjs,cjs,ts}"] },
+  { languageOptions: { globals: globals.browser } },
+  { ignores: ["**/node_modules/", "dist/"] },
+  pluginJs.configs.recommended,
+  ...tseslint.configs.recommended,
+];
```

Comment:
`eslint-plugin-prettier` since you're using prettier, and `eslint-plugin-perfectionist` would be nice also. 

See this [config from core](https://github.com/solidatus/solidatus/blob/master/test/Solidatus.ApiTest/eslint.config.mjs) for a filled out lint config, we really ought to publish as a package and include from our other repos...